### PR TITLE
Scripts/EasternKingdoms Every quest, every spawn working in Redridge …

### PR DESCRIPTION
**Pull request by AussieGlenn**
…mountains.

**Changes proposed:**

Fix every quest in Redridge mountains
Fix overspawns/underspawns
Repair Trainers/Vendors
Add pathing

**Issues addressed:** Closes #  (insert issue tracker number)

**Tests performed:** (Does it build, tested in-game, etc.)
Full quest chain with 2 different characters

**Known issues and TODO list:** (add/remove lines as needed)

- [ ] 
- [ ]

### CODE REVIEW

**Comment by AussieGlenn** [comment]
Hi Traesh,

Finally.

There are 3 hacked quests, anyone wanting to fix these good luck to them.

It will be multiplayer safe because I bypassed all the auras to make sure it worked properly after logout and with multiple players.

Feel free to change whatever you feel like.

Thanks,

Glenn

File: src/server/scripts/EasternKingdoms/zone_redridge_mountains.cpp [hunk 1]
```diff
@@ -379,10 +385,110 @@ class npc_redridge_citizen : public CreatureScript
...
+public:
+    npc_keeshan_riverboat() : CreatureScript("npc_keeshan_riverboat") { }
+
+    struct npc_keeshan_riverboatAI : public VehicleAI
+    {
+        npc_keeshan_riverboatAI(Creature* creature) : VehicleAI(creature)
+        {
+        }
+
+        void UpdateAI(uint32 diff) override
```
Comment (Nefertumm) [inline src/server/scripts/EasternKingdoms/zone_redridge_mountains.cpp]: Travis here, comment the diff

**Comment by Traesh** [comment]
Nice work as always, i'm gonna test that :)

Can you give me the 3 hackfixed quests ids ?

**Comment by AussieGlenn** [comment]
Hi Traesh,

https://www.wowhead.com/quest=26520/saving-foreman-oslow

There is a script for it, but it took 4 days of bashing before i gave up, so now it autocompletes on touching the orb

https://www.wowhead.com/quest=26616/its-never-over

I have the areatrigger id, matched up coords, linked in areatrigger_involvedrelation, cast the spell to complete the quest, but no banana.

https://www.wowhead.com/quest=26668/detonation

This involves playing a movie that shows the phased Redridge mountains blowing up. Was not in the sniffs i had, and since i did not use phases in my solutions, it probably would not show what was intended, so it autocompletes.

Thanks,

Glenn

**Comment by AussieGlenn** [comment]
Hi @DrThum ,

yes, you are indeed correct, how did that happen?

@Traesh could you fix when/if you commit?

Thanks,

Glenn

**Pull request merged**